docs/data_driven_testing.adoc

```diff
@@ -99,7 +99,7 @@ Condition not satisfied:
 
 Math.max(a, b) == c
     |    |  |  |  |
-    |    7  0  |  7
+    |    7  4  |  7
```

Comment:
@szpak sure, it's because this shows the result of the test [MathSpec.groovy](https://github.com/spockframework/spock/blob/master/spock-specs/src/test/groovy/org/spockframework/docs/datadriven/v3/MathSpec.groovy), which have 4 set for b, not 0. See sections _Syntactic Variations_ and _Reporting of Failures_ in [docs](http://spockframework.org/spock/docs/1.1-rc-3/data_driven_testing.html) and you'll see, that it's confusing for the reader of that docs ;)